src/app/tests/TestSceneTable.cpp

```diff
@@ -152,6 +156,43 @@ static uint32_t OO_buffer_serialized_length = 0;
 static uint32_t LC_buffer_serialized_length = 0;
 static uint32_t CC_buffer_serialized_length = 0;
 
+// clang-format off
+static const MockNodeConfig SceneMockNodeConfig({
+    MockEndpointConfig(kTestEndpoint1, {
+        MockClusterConfig(kScenesClusterId, {}),
+        MockClusterConfig(kOnOffClusterId, {
+            kOnOffAttId
```

Comment:
could we use actual attribute IDs? Like `chip::app::Clusters::OnOff::Attributes::OnOff::Id` ? maybe with a `using` clause to make these shorter?

Or are we intentionally using mock values (in which case maybe we should use MockAttributeId)